contracts/scripts/util/arbitrum_gas_util.ts

```diff
@@ -0,0 +1,291 @@
+const fs = require('fs');
+/*
+ Web3 needs to be used over Ethers since web3 tx receipts
+ for Arbitrum correctly store the feeStats object.
+ Ethers' tx receipts aren't updated to store that yet.
+ */
+const Web3 = require('web3');
+require('dotenv').config();
+
+// current git commit
+const REVISION = require('child_process')
+    .execSync('git rev-parse HEAD')
+    .toString().trim();
+
+const web3Mainnet = new Web3(process.env.ARBITRUM_URL);
+const web3Rinkeby = new Web3(process.env.ARBITRUM_TESTNET_URL);
+
+const ARBITRUM_GAS_COST_FILE = './ArbitrumGasCosts.md';
+const NUM_DECIMALS = 7;
+/*
+ This contract was predeployed to Arbitrum and can be used
+ to retreive current gas prices.
+ For more info on pre-deploys: https://developer.offchainlabs.com/docs/useful_addresses
+ */
+const MAINNET_ARB_GAS_INFO_ADDRESS =
+    "0x000000000000000000000000000000000000006C";
+
+export async function processGasResultsToFile(
+    blsTxHash, normalTxHash, numTxsAggregated) {
+        console.log("Retrieving gas results"
+    );
+
+    const blsEntry = await newTableEntry(
+        blsTxHash, numTxsAggregated, true
+    );
+    const normalEntry = await newTableEntry(
+        normalTxHash, numTxsAggregated, false
+    );
+
+    const input = '\n' + blsEntry + '\n' + normalEntry;
+
+    console.log("Writing gas results to file");
+    fs.appendFileSync(ARBITRUM_GAS_COST_FILE, input);
```

Comment:
Non-blocking i/o ops preferred.
```
import { appendFile } from 'fs/promises';
...
```
```suggestion
    await fs.appendFile(ARBITRUM_GAS_COST_FILE, input);
```